common.sh

```diff
@@ -1,8 +1,12 @@
 #!/bin/bash
 
+if [[ ! $(command -pv go) ]]; then
+    export PATH="/usr/local/go/bin:$PATH"
+fi
+
 eval "$(go env)"
 
-export PATH="${GOPATH}/bin:/usr/local/go/bin:$PATH"
+export PATH="${GOPATH}/bin:$PATH"
```

Comment:
Do we need the conditional here, probably we should just always export PATH="/usr/local/go/bin:$PATH" before running go env?

Note that this will still fail on a clean install the very first time the 01 script gets run...

Reply:
At least it should not fail later when the *go* is installed